recipes/cmaes/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import (
+    export_conandata_patches,
+    get,
+    rmdir,
+    rm,
+    copy,
+    apply_conandata_patches,
+)
+from conan.tools.build import check_min_cppstd
+import os
+
+
+class CmaesConan(ConanFile):
+    name = "cmaes"
+
+    generators = "CMakeDeps"
+
```

Comment:
```suggestion
```

Avoid mixing attributes when generate(), it may difficult when reviewing, and also may confuse users.